### PR TITLE
[14.0][MIG] attribute set completeness xbo

### PR DESCRIPTION
**Pull request by xavier-bouquiaux**
@rousseldenis , @Cedric-Pigeon 

test ok

### CODE REVIEW

File: attribute_set_completeness/models/attribute_set_owner_mixin.py [hunk 1]
```diff
@@ -19,7 +19,7 @@ class AttributeSetOwnerMixin(models.AbstractModel):
         related="attribute_set_id.attribute_set_completeness_ids", readonly=True,
     )
     attribute_set_not_completed_ids = fields.Many2many(
-        comodel_name="attribute.set.completeness", compute="_compute_completion_rate",
+        comodel_name="attribute.set.completeness", readonly=True
```
Comment (rousseldenis) [inline attribute_set_completeness/models/attribute_set_owner_mixin.py]: Why changing the field to a non computed one ?

The here under _compute_completion_rate method is still filling it...

**Review by rousseldenis — CHANGES_REQUESTED** [review]
LGTM apart the changed computed field.

@xavier-bouquiaux

File: attribute_set_completeness/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,17 @@
+# Copyright 2021 ACSONE SA/NV
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    "name": "Attribute Set Completeness",
+    "version": "14.0.1.0.0",
+    "license": "AGPL-3",
+    "author": "ACSONE SA/NV",
+    "website": "https://github.com/shopinvader/odoo-pim",
```
Comment (rousseldenis) [inline attribute_set_completeness/__manifest__.py:9]:
```suggestion
    "website": "https://github.com/OCA/odoo-pim",
```

File: attribute_set_completeness/__manifest__.py [hunk 3]
```diff
@@ -0,0 +1,17 @@
+# Copyright 2021 ACSONE SA/NV
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    "name": "Attribute Set Completeness",
+    "version": "14.0.1.0.0",
+    "license": "AGPL-3",
+    "author": "ACSONE SA/NV",
```
Comment (ivantodorovich) [inline attribute_set_completeness/__manifest__.py:8]:
```suggestion
    "author": "ACSONE SA/NV, Odoo Community Association (OCA)",
```

**Review by ivantodorovich — CHANGES_REQUESTED** [review]

File: attribute_set_completeness/__manifest__.py [hunk 4]
```diff
@@ -1,9 +1,9 @@
-# Copyright 2020 ACSONE SA/NV
+# Copyright 2021 ACSONE SA/NV
```
Comment (simahawk) [inline attribute_set_completeness/__manifest__.py:1]: 2020 is correct

File: attribute_set_completeness/components/attribute_set_owner_event_listener.py [hunk 5]
```diff
@@ -1,4 +1,4 @@
-# Copyright 2020 ACSONE SA/NV (<http://acsone.eu>)
+# Copyright 2021 ACSONE SA/NV (<http://acsone.eu>)
```
Comment (simahawk) [inline attribute_set_completeness/components/attribute_set_owner_event_listener.py:1]: 2020 everywhere unless you added new files

File: attribute_set_completeness/models/attribute_set_owner_mixin.py [hunk 6]
```diff
@@ -14,15 +14,19 @@ class AttributeSetOwnerMixin(models.AbstractModel):
...
     )
 
     attribute_set_completeneness_ids = fields.One2many(
         related="attribute_set_id.attribute_set_completeness_ids", readonly=True,
     )
     attribute_set_not_completed_ids = fields.Many2many(
-        comodel_name="attribute.set.completeness", compute="_compute_completion_rate",
+        comodel_name="attribute.set.completeness",
+        compute="_compute_completion_rate",
+        store=True,
```
Comment (simahawk) [inline attribute_set_completeness/models/attribute_set_owner_mixin.py:26]: store for?

**Review by simahawk — CHANGES_REQUESTED** [review]

**Comment by simahawk** [comment]
replaced by https://github.com/OCA/odoo-pim/pull/98

**Pull request closed without merging**